user-guide/Troubleshooting/Procedures/Collecting_data_to_report_an_issue_to_TechSupport.md

```diff
@@ -0,0 +1,67 @@
+---
+uid: Collecting_data_to_report_an_issue_to_TechSupport
+---
+
+# Collecting data to report an issue to Technical Support
+
+The type of data required depends on the type of issue you are experiencing:
+
+- For all issues: [Log Collector packages](#log-collector-packages)
+
+- For Cube-related issues: [Cube Debug information](#cube-debug-information)
+
+- For element or service-related issues: [DELT export packages](#delt-export-packages)
+
+- For element communication issues: [Stream Viewer traces](#stream-viewer-traces)
+
+## Log Collector packages
+
+A Log Collector package provides comprehensive information about the DataMiner System, including software version, cluster composition, network and database statistics, and more. Even if the Log Collector package appears empty, it can be valuable information, helping Technical Support verify that the DataMiner core logic is functioning correctly, and the issue lies elsewhere.
+
+To save a Log Collector package:
+
+1. Connect to the DataMiner server using Remote Desktop.
+
+1. Run the Log Collector tool by following the steps in [SLLogCollector](xref:SLLogCollector).
+
+1. The Log Collector package is saved by default in an *SL_LogCollector* folder on your desktop.
+
+## Cube Debug information
+
+### Cube is connected to your DMA
+
+1. In DataMiner Cube, go to *Apps* > *About*.
+
+1. In the *general* tab, select *Export debug information*.
```

Comment:
The remark above about screenshots applies here as well. An image is more intuitive and leaves less room for misunderstanding.